library/src/main/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneFlight.java

```diff
@@ -47,18 +52,43 @@ private void addCreateSteps(LandingZoneFlightBeanBag flightBeanBag, FlightMap in
         new GetBillingProfileStep(flightBeanBag.getBpmService()), RetryRules.shortExponential());
 
     if (!requestedLandingZone.isAttaching()) {
-      addStep(
-          new CreateLandingZoneResourcesFlightStep(
-              flightBeanBag.getLandingZoneService(),
-              requestedLandingZone,
-              LandingZoneFlightMapKeys.CREATE_LANDING_ZONE_RESOURCES_INNER_FLIGHT_JOB_ID));
-      addStep(
-          new AwaitCreateLandingZoneResourcesFlightStep(
-              LandingZoneFlightMapKeys.CREATE_LANDING_ZONE_RESOURCES_INNER_FLIGHT_JOB_ID));
+      var stepsDefinitionProvider =
```

Comment:
This is just the logic that was in the `CreateLandingZoneResourcesFlight`, minus the `ArmManagers` initialization (which is now its own step).